docs/index.rst

```diff
@@ -3,26 +3,37 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+========
 Welcome!
 ========
 
-Welcome to the Contributors Portal for Access to Memory (AtoM), Archivematica and Enduro.
+Welcome to the Contributors Portal for Access to Memory (AtoM),
+Archivematica and Enduro.
+
+At `Artefactual <https://www.artefactual.com>`__, we are currently
+working on improving the contribution process, and we deemed useful
+to have a Portal to regroup relevant information for our main
+software projects.
 
-This Portal was first created in May 2024.
+This Portal was first created in May 2024. It is maintained as a 
+`GitHub repo <https://www.github.com/artefactual/artefactual.github.io>`__.
+
+Suggestions for this Portal are welcome - you can either `open an issue <https://www.github.com/artefactual/artefactual.github.io/issues>`__
+or `send us an email <mailto:contributors@artefactual.com>`__.
 
 .. toctree::
    :maxdepth: 1
    :caption: Contents:
    
    cheatsheet.rst
+   dependencies.rst
    i18n.rst
    
-Why this Portal
----------------
-
-At Artefactual, we are currently working on improving the contribution process, and we deemed useful to have a Portal to help regroup relevant information for our main software projects.
+The :doc:`Cheatsheet <cheatsheet>`, is a table of links for each
+of the projects we maintain.
 
-What is the Portal
-------------------
+The page :doc:`Dependencies <dependencies>` is a set of notes on
```

Comment:
I'd suggest saying "The Dependencies page" rather than "The page Dependencies"

Also, non-developers and/or new developers may find this sentence unclear, since it mainly just repeats the word dependencies without explaining what that is. I would suggest instead something that defines it a bit more? For example: 


```
The :doc:`Dependencies <dependencies>` page is a list of external projects and code libraries used by the applications we maintain. 
```